server/src/main/java/org/opentosca/toscana/plugins/kubernetes/docker/mapper/MapperUtils.java

```diff
@@ -0,0 +1,31 @@
+package org.opentosca.toscana.plugins.kubernetes.docker.mapper;
+
+import java.util.Comparator;
+import java.util.Optional;
+
+import org.opentosca.toscana.model.capability.OsCapability;
+import org.opentosca.toscana.plugins.kubernetes.docker.mapper.model.DockerImageTag;
+
+public class MapperUtils {
+
+    public static final Comparator<DockerImageTag> TAG_COMPARATOR_MINOR_VERSION =
+        Comparator.comparing(DockerImageTag::toVersion);
+
+    public static boolean isAnyOptionalPresent(Optional... optionals) {
+        for (Optional optional : optionals) {
+            if (optional.isPresent()) {
+                return true;
+            }
+        }
+        return false;
+    }
```

Comment:
Could be replaced with `        Arrays.stream(optionals).anyMatch(optional -> optional.isPresent());`